index.js

```diff
@@ -1,20 +1,26 @@
 /* jshint node: true */
 'use strict';
 
-const REFLECT_JS_VERSION = 'latest',
-      REFLECT_JAVASCRIPT = 'https://cdn.reflect.io/' + REFLECT_JS_VERSION +'/reflect.js',
-      DEFAULT_CSS_SOURCE = 'https://cdn.reflect.io/' + REFLECT_JS_VERSION + '/reflect.css';
+const DEFAULT_ASSET_VERSION = 'latest';
+const getJSSource = ver => `https://cdn.reflect.io/${ver}/reflect.js`;
+const getCSSSource = ver => `https://cdn.reflect.io/${ver}/reflect.css`;
 
 const getHead = function(config) {
-  var cssSource;
+  let cssSource = getCSSSource(DEFAULT_ASSET_VERSION);
+  let jsSource = getJSSource(DEFAULT_ASSET_VERSION);
 
-  if (config.reflect && config.reflect.css) {
-    cssSource = config.reflect.css;
-  } else {
-    cssSource = DEFAULT_CSS_SOURCE;
+  if (config.reflect) {
+    if (config.reflect.version) {
+      jsSource = getJSSource(config.reflect.version);
+      cssSource = getCSSSource(config.reflect.version);
+    }
+
+    if (config.reflect.css) {
```

Comment:
Do you think it makes sense to also add config.reflect.js here in case they'd like to target another version... say in s3?